corehq/apps/analytics/tasks.py

```diff
@@ -415,16 +415,25 @@ def _get_report_count(domain):
     return get_report_builder_count(domain)
 
 
+def _log_failed_periodic_data(email, message):
+    soft_assert(to='{}@{}'.format('bbuczyk', 'dimagi.com'))(
+        False, "ANALYTICS - Failed to sync periodic data", {
+            'user_email': email,
+            'message': message,
+        }
+    )
+
+
 @periodic_task(run_every=crontab(minute="0", hour="4"), queue='background_queue')
 def track_periodic_data():
     """
     Sync data that is neither event or page based with hubspot/Kissmetrics
     :return:
     """
     # Start by getting a list of web users mapped to their domains
-    six_months_ago = date.today() - timedelta(days=180)
+    two_days_ago = date.today() - timedelta(days=2)
```

Comment:
I think this is going to miss a ton of users (especially once we fix the session bug), since we only require users to log in every two weeks (or maybe its only if you are inactive for two weeks which would mean even more users missed), and this checks login date not last activity on the site.